lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -55,31 +56,43 @@ class LinkBlockingFirstPaintAudit extends Audit {
       };
     }
 
-    let totalSpendTime = 0;
-    const results = artifact.items
-      .filter(item => item.tag.tagName === tagFilter)
-      .map(item => {
-        totalSpendTime += item.spendTime;
+    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
 
-        return {
-          url: item.tag.url,
-          label: `delayed first paint by ${item.spendTime}ms`
-        };
-      });
+    let startTime = Number.MAX_VALUE;
+    let endTime = 0;
+    startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), startTime);
```

Comment:
would this work:

`let startTime = filtered.reduce((t, item) => Math.min(t, item.startTime), Number.MAX_VALUE);`